modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/exportimport/data/handler/OrganizationStagedModelDataHandler.java

```diff
@@ -345,6 +365,33 @@ private void _importAddresses(
 			importedOrganization.getOrganizationId(), addresses);
 	}
 
+	private Organization _importCountry(
+			PortletDataContext portletDataContext, Organization organization)
+		throws Exception {
+
+		List<Element> countryElements =
+			portletDataContext.getReferenceDataElements(
+				organization, Country.class);
+
+		if (!countryElements.isEmpty()) {
```

Comment:
Can we replace this with a loop like the other methods? I know that in this case it's only once but because the `portletDataContext.getReferenceDataElements` returns always a list we can iterate on it (in this case we avoid also to do the get(0) code